### PR TITLE
feat(create group): When creating a group, the generated id is now editable

### PR DESCRIPTION
**Pull request by tommydeboer**
This is pending on this PR: https://github.com/molgenis/molgenis/pull/8891

### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [x] Code unit/integration/system tested
- [x] User documentation updated
- [x] Conventional commits (squash if needed)
- [x] No warnings during install
- [x] Updated javascript typing

### CODE REVIEW

File: packages/security/src/components/GroupCreate.vue [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
           <div class="form-group">
             <label for="groupIdentifierInput">{{'security-ui-group-attribute-name-name' | i18n}}</label>
-            <input v-model="groupIdentifier" readonly type="text" class="form-control" id="groupIdentifierInput"
+            <input v-model="groupIdentifier" type="text" class="form-control" id="groupIdentifierInput"
```
Comment (connoratrug) [inline packages/security/src/components/GroupCreate.vue]: the edited identifier is not validated, both for correctness and uniqueness

**Comment by connoratrug** [comment]
please merge PR with master (needed due to CI/Jenkins update)

File: packages/security/config/index.js [hunk 2]
```diff
@@ -23,10 +23,12 @@ module.exports = {
     assetsPublicPath: '/',
     proxyTable: {
       '/login': {
-        target: 'http://localhost:8080'
+        target: 'https://latest.dev.molgenis.org',
```
Comment (fdlk) [inline packages/security/config/index.js]: is dead, choose between master.dev and latest.test

File: packages/security/src/components/GroupCreate.vue [hunk 3]
```diff
@@ -104,12 +112,14 @@
...
       },
-
-      checkGroupName: _.throttle(function () {
-        const pipesRegEx = '/-/g'
-        const packageName = this.groupIdentifier.replace(pipesRegEx, '_')
-        this.$store.dispatch('checkRootPackageExists', packageName).then((exists) => {
-          this.isGroupNameAvailable = !exists
+      checkGroupIdentifier: _.throttle(function () {
+        this.isGroupIdentifierValid = true
+        if (!this.groupIdentifier.match(/^[a-zA-Z0-9_-]+$/)) {
```
Comment (connoratrug) [inline packages/security/src/components/GroupCreate.vue]: Please describe the regex , what is it checking

File: packages/security/config/index.js [hunk 4]
```diff
@@ -23,10 +23,12 @@ module.exports = {
     assetsPublicPath: '/',
     proxyTable: {
       '/login': {
-        target: 'http://localhost:8080'
+        target: 'https://latest.test.molgenis.org',
```
Comment (dennishendriksen) [inline packages/security/config/index.js]: accidental commit?

File: packages/security/config/index.js [hunk 5]
```diff
@@ -23,10 +23,12 @@ module.exports = {
...
+        changeOrigin: 'true'
       },
       '/api': {
-        target: 'http://localhost:8080'
+        target: 'https://latest.test.molgenis.org',
```
Comment (dennishendriksen) [inline packages/security/config/index.js]: accidental commit?

File: packages/security/src/components/GroupCreate.vue [hunk 6]
```diff
@@ -76,20 +79,34 @@
...
+        groupIdentifierTaken: null
       }
     },
     computed: {
-      groupIdentifier () {
+      groupIdentifierSlug () {
         return slugService.slugify(this.groupName)
+      },
+      invalidGroupIdentifier () {
+        return this.groupIdentifier && !this.groupIdentifier.match(/^[\w-]+$/)
```
Comment (connoratrug) [inline packages/security/src/components/GroupCreate.vue]: please add comment describing the regex in human readable form, and unit test it

Comment (fdlk) [reply packages/security/src/components/GroupCreate.vue]: I think this is simple enough as regexes go. Word character or minus. It gets tested in the unit tests with both correct and incorrect group id

Comment (connoratrug) [reply packages/security/src/components/GroupCreate.vue]: i agree it's simple enough as regexes go, however the all regex are hard to read as code go's, add a comment seems like a good solution to me.

Comment (fdlk) [reply packages/security/src/components/GroupCreate.vue]: I've added a link to documentation page with examples

**Pull request merged**